packages/rainbowkit/src/core/network/enhancedProvider.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from 'vitest';
+import { enhancedProviderHttp } from './enhancedProvider';
+
+describe.skip('createHttpClient', () => {
```

Comment:
Will do a follow up PR to fix this unit test. Faces issues with github secrets and `process.env`, but we can run this locally for now.